charmdir.go

```diff
@@ -316,3 +336,24 @@ func checkFileType(path string, mode os.FileMode) error {
 	}
 	return fmt.Errorf(e, path)
 }
+
+// MaybeCreateVersionFile creates/overwrite charm version file.
+func MaybeGenerateVersionString(path string) (string, error) {
```

Comment:
wrong comment
also, we don't need to export this method

Reply:
done

Reply:
i was finding issue. so i am still exporting. will try again tomorrow.

Reply:
I faced issues not exporting to be used in tests. Maybe i will again take a look.